ghost/members-api/lib/repositories/member.js

```diff
@@ -337,10 +340,10 @@ module.exports = class MemberRepository {
                 }
             }
         }
-
+        const updatedAttribution = this._memberAttributionService.updateAttributionForSource(data.attribution, source);
```

Comment:
I think this is happening to close to the core of the code, and would be better suited to living at the endpoint/BREAD service level

Reply:
Moved to https://github.com/TryGhost/Ghost/pull/15499